packages/support/src/settings/_variables.color.docs.scss

```diff
@@ -117,6 +117,36 @@ Markup:
 Style guide: style.color.status
 */
 
+/*
+Focus colors
```

Comment:
What do you think about just putting the focus colors in the "Additional colors" section rather than its own? I'm thinking how there are other state colors like `$color-visited` which should also fall into the same category as the focus colors, and wouldn't necessarily deserve their own section